sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractReceiptOperation.cs

```diff
@@ -71,13 +72,24 @@ private async Task<Response> UpdateStatusAsync(bool async, CancellationToken can
 
                     // TODO: When they support extracting more than one receipt, add a pageable method for this.
                     // https://github.com/Azure/azure-sdk-for-net/issues/10389
-                    _value = new ExtractedReceipt(update.Value.AnalyzeResult.DocumentResults.First(), update.Value.AnalyzeResult.ReadResults.First());
+                    //_value = new ExtractedReceipt(update.Value.AnalyzeResult.DocumentResults.First(), update.Value.AnalyzeResult.ReadResults.First());
+                    _value = ConvertToExtractedReceipts(update.Value.AnalyzeResult.DocumentResults, update.Value.AnalyzeResult.ReadResults);
                 }
 
                 _response = update.GetRawResponse();
             }
 
             return GetRawResponse();
         }
+
+        private static IReadOnlyList<ExtractedReceipt> ConvertToExtractedReceipts(IList<DocumentResult_internal> documentResults, IList<ReadResult_internal> readResults)
+        {
+            List<ExtractedReceipt> receipts = new List<ExtractedReceipt>();
+            for (int i = 0; i < documentResults.Count; i++)
+            {
+                receipts.Add(new ExtractedReceipt(documentResults[i], readResults[i]));
```

Comment:
will this add all the pages of the readResult on each ExtractedReceipt, or only the pages matching the documentResult?

Reply:
I've updated it - it should only add the pages matching the page range in documentResult.

Reply:
Tracking with issue: https://github.com/Azure/azure-sdk-for-net/issues/10744